bot/seasons/valentines/movie_generator.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import random
+from os import environ
+from urllib import parse
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class MovieGenerator:
+    """
+    a cog that gives random romance movie suggestion to a user
```

Comment:
Grammatical corrections
```suggestion
    A cog that returns a random romance movie suggestion to a user
```